lib/packages/docutils/rstast.nim

```diff
@@ -92,9 +96,15 @@ type
       level*: int             ## level of headings starting from 1 (main
                               ## chapter) to larger ones (minor sub-sections)
                               ## level=0 means it's document title or subtitle
-    of rnFootnote, rnCitation, rnFootnoteRef, rnOptionListItem:
+    of rnFootnote, rnCitation, rnOptionListItem:
       order*: int             ## footnote order (for auto-symbol footnotes and
                               ## auto-numbered ones without a label)
+    of rnRef, rnSubstitutionReferences,
+        rnInterpretedText, rnField, rnInlineCode, rnCodeBlock:
+      li*: TLineInfo          ## To have line/column info for warnings at
```

Comment:
`li` is neither standard in nim sources nor self documenting; instead use `info` which is at least standard in nim compiler

Reply:
thank you, done.